R/modify_db.R

```diff
@@ -191,3 +191,57 @@ calc_fram_scaling <- function(fram_db, table_name, df) {
 
   return(df_mod)
 }
+
+#'  `r lifecycle::badge("experimental")`
+#' Zero out fishery catch for selected fisheries
+#'
+#' Nukes a fishery for one or more timesteps; useful to streamline scenario modeling during the NoF process. Leaves FisheryFlag alone, and sets scaler, quota, msf scaler, and msf quota to 0. If using `zero_fishery()` on a database that is loaded into FRAM, remember to reload the database before doing more modeling!
```

Comment:
Can we have the default this to zeroing out Puget Sound marine fisheries or whatever @mawalden needs? In practice this primarily going to be used to measure the impact of our marine fisheries on Puget Sound abundances for SKRW work.